### DISCUSSION TITLE
Rules not triggered when bulk updating transactions

### DISCUSSION THREAD

**Discussion by thomase1234** [firefly-iii/firefly-iii#6503] [Firefly III] [answered]
Hi all, Thanks @JC5 for this great product. I've been using only for some days, but it's already helping me to get a good understanding of my income and costs streams.

I do have a question related to the Rules handling. I created rules which trigger whenever a "transaction is updated". This works as expected and those rules are evaluated when f.e. I add a tag to a single transaction.

However, when I select one or more transaction in the transaction list, click "Edit selected in Bulk" and add the same tag. The tag gets added to the selected transactions, but the rules are not evaluated. I would have expected that every update to a transaction would trigger the evaluation of those rules.

Is this expected behavior ?

Thanks for your assistance,
Thomas

**Comment by JC5** [comment, marked as answer]
Thanks! That's a bug, and it'll be fixed in the new release :+1:

**Comment by thomase1234** [comment]
5.7.13 was released yesterday. I've installed that version, but the problem still remains. I guess the fix didn't make it into this version?

**Reply by JC5** [reply]
It should have. Unless I forgot. Ill check later for you.

**Reply by JC5** [reply]
Nope I forgot lol

**Reply by thomase1234** [reply]
No issue. At least I know I don't have to try any further. Thanks anyway. Great tool!

**Comment by thomase1234** [comment]
I can confirm it's fixed in 5.7.14 which was just released. 

Thanks a lot!
